lib/esm-utils.js

```diff
@@ -25,7 +30,7 @@ const requireOrImport = async file => {
 exports.loadFilesAsync = async (files, preLoadFunc, postLoadFunc) => {
   for (const file of files) {
     preLoadFunc(file);
-    const result = await requireOrImport(file);
+    const result = await exports.requireOrImport(path.resolve(file));
```

Comment:
If the file in question is a package name, `path.resolve` will fail (AFAIK with an exception). Why did you add a `path.resolve` here?

Reply:
Oh, I see. `--require`-d files can be module names, and so you didn't want `requireOrImport`, which is now used also in `handleRequire`, to resolve them to a path. So you moved it to `loadFileAsync`, where it _is_ needed (for Windows, it seems...)